wiki/quest-modding.md

```diff
@@ -155,7 +155,17 @@ You will need to temporarily remove all secondary accounts before modding the ga
 You can add them back once the modding process is completed.
 :::
 
+:::tip NOTE
```

Comment:
probably better to copy the whole downgrade section to here instead of forwarding to the bmbf page